esp-storage/Cargo.toml

```diff
@@ -40,25 +40,25 @@ critical-section = []
 ## Bytewise read emulation
 bytewise-read = []
 
-## Enable logging output using `defmt` and implement `defmt::Format` on certain types.
+## Implement `defmt::Format` on certain types.
 defmt = ["dep:defmt"]
 
 #! ### Chip selection
 #! One of the following features must be enabled to select the target chip:
 
-##
+## Target the ESP32-C2.
```

Comment:
I'm personally not a fan of this, it's just more words to describe what's I think is obvious:

<img width="596" height="352" alt="Image" src="https://github.com/user-attachments/assets/7b69281c-5061-45e0-a563-32e6d15f3d1c" />